apps/financial-aid/backend/src/app/modules/staff/staff.service.ts

```diff
@@ -72,6 +74,26 @@ export class StaffService {
     })
   }
 
+  async createAdmin(
+    input: CreateStaffDto,
+    municipalityInput: CreateMunicipalityDto,
+    t: Transaction,
+  ): Promise<StaffModel> {
+    return await this.staffModel.create(
+      {
+        nationalId: input.nationalId,
+        name: input.name,
+        municipalityId: municipalityInput.municipalityId,
+        email: input.email,
+        roles: input.roles,
```

Comment:
Should we maybe use `CreateStaff` function instead?

Reply:
We already have a `CreateStaff` function, so keep the name or change ? 🙃

Reply:
No use that instead of creating a new one 👼